src/Backoffice/Users/Domain/DataTransferObjects/pVerifyEligibilitySummaryRequestDTO.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Users\Domain\DataTransferObjects;
+
+class pVerifyEligibilitySummaryRequestDTO
+{
+    public function __construct(
+        public string $payerCode,
+        public string $payerName,
+        public string $Provider_LastName,
+        public string $Provider_NPI,
+        public string $Provider_PIN,
+        public string $Patient_First,
+        public string $Patient_DOB,
+        public string $Patient_Last,
+        public string $memberID,
+        public string $Date_Of_Service,
+        public string $referenceId,
+        public string $Location,
+        public string $PracticeTypeCode,
+        public string $IncludeTextResponse,
+        public string $InternalId,
+        public string $CustomerID,
+    ) {
+    }
+
+    public function toArray(): array
```

Comment:
what do we need this for?

Reply:
to build te request body in the saloon request class once we know what should go in the body.